uint8[22] angles #numOfJoints
uint32 timeStamp
float32 cycleTime
